src/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -143,6 +143,13 @@ function should_hack_data_sync_retry() {
   return 1
 }
 
+function generate_postgresql_libraries_config() {
+  if [ -v POSTGRESQL_LIBRARIES ]; then
```

Comment:
```suggestion
  if [ -v ${POSTGRESQL_LIBRARIES} ]; then
```

Reply:
Or even better way: `if [ -z ${var+x} ]; then echo "var is unset"; else echo "var is set to '$var'"; fi`